x-pack/plugins/maps/common/constants.ts

```diff
@@ -144,6 +144,7 @@ export enum DRAW_SHAPE {
   LINE = 'LINE',
   SIMPLE_SELECT = 'SIMPLE_SELECT',
   DELETE = 'DELETE',
+  INSERT = 'INSERT',
```

Comment:
`INSERT` is misleading name since its also used for deleting. How about changing to `WAIT`?